.chloggen/feat_faster-vcenter-vm-network-calls.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: vcenterreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Changes the method for collecting VMs used by the `vccenterreceiver` to the more time efficient `CreateContainerView` method."
```

Comment:
I would modify this a bit. really we are using a container view's "retrieve" method now